src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -34,14 +35,17 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PutMapping("/movie-sessions")
-    public void addToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
-        shoppingCartService.addSession(
-                movieSessionService.get(movieSessionId), userService.get(userId));
+    public void addToCart(Authentication authentication, @RequestParam Long movieSessionId) {
+        shoppingCartService.addSession(movieSessionService
+                        .get(movieSessionId),
+                userService.findByEmail(authentication.getName())
+                        .orElseThrow(() -> new RuntimeException("Shopping cart wasn't found.")));
```

Comment:
```suggestion
                        .orElseThrow(() -> new RuntimeException("User with this email wasn't found.")));
```
same for other case. This is the method of userService)